src/_guides/language/language-tour.md

```diff
@@ -3721,18 +3704,11 @@ Although an async function might perform time-consuming operations,
 it returns immediately—before
 any of its body executes.
 
-<?code-excerpt "language-tour/async-await/bin/main.dart (checkVersion-lookUpVersion)"?>
-{% prettify dart %}
-Future checkVersion() async {/* ... */}
-
-Future lookUpVersion() async {/* ... */}
-{% endprettify %}
-
```

Comment:
I removed the code excerpt because, IMHO, it did nothing to support the prose.

Reply:
We need to add a "Dart 2.0 note" here about this behavior changing, as described in https://github.com/dart-lang/sdk/blob/master/docs/newsletter/20170915.md#synchronous-async-start.

Reply:
Note (and link) added.